osu.Game/Screens/Menu/IntroWelcome.cs

```diff
@@ -0,0 +1,163 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osuTK;
+using osu.Framework.Allocation;
+using osu.Framework.Audio;
+using osu.Framework.Audio.Sample;
+using osu.Framework.Screens;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Textures;
+using osuTK.Graphics;
+
+namespace osu.Game.Screens.Menu
+{
+    public class IntroWelcome : IntroScreen
+    {
+        protected override string BeatmapHash => "64E00D7022195959BFA3109D09C2E2276C8F12F486B91FCF6175583E973B48F2";
+        protected override string BeatmapFile => "welcome.osz";
+        private const double delay_step_two = 2142;
+        private SampleChannel welcome;
+        private SampleChannel pianoReverb;
+
+        [BackgroundDependencyLoader]
+        private void load(AudioManager audio)
+        {
+            Seeya = audio.Samples.Get(@"Intro/welcome/seeya");
+
+            if (MenuVoice.Value)
+            {
+                welcome = audio.Samples.Get(@"Intro/welcome/welcome");
+                pianoReverb = audio.Samples.Get(@"Intro/welcome/welcome_piano");
+            }
+        }
+
+        protected override void LogoArriving(OsuLogo logo, bool resuming)
+        {
+            base.LogoArriving(logo, resuming);
+
+            if (!resuming)
+            {
+                welcome?.Play();
+                pianoReverb?.Play();
+
+                Scheduler.AddDelayed(delegate
```

Comment:
`delegate` is a really old thing now. Lambdas are a bit more hip and modern nowadays.

```suggestion
                Scheduler.AddDelayed(() =>
```

Reply:
Yeah you're right! That was just a blind copy-paste from `IntroCircles` really. I'll include this with the change for the drawable lifetimes.